aten/src/ATen/native/mps/operations/Quantized.mm

```diff
@@ -295,15 +749,32 @@ Tensor _weight_int8pack_mm_mps(const Tensor& A, const Tensor& B, const Tensor& s
       }
 #endif
       id<MTLComputeCommandEncoder> computeEncoder = mpsStream->commandEncoder();
-      const std::string kernel = fmt::format("int8pack_mm_{}", scalarToMetalTypeString(A));
+      std::string kernel;
+      if (M == 1) {
+        kernel = fmt::format("int8pack_mv_{}", scalarToMetalTypeString(A));
+      } else if (M < 4) {
+        kernel = fmt::format("int8pack_mm_{}", scalarToMetalTypeString(A));
```

Comment:
So for M<4 we keep using ATen kernel?

Reply:
Yes

Reply:
How much perf would we loose if we are to use mv kernel for those cases? (Just call it M times)
It would be good to have two kernels (mv + mm) rather than 3

Reply:
I can do a follow-up to clean this up.